yt/yt/server/query_tracker/proxy_service.cpp

```diff
@@ -106,6 +109,9 @@ class TProxyService
         auto queryId = FromProto<TQueryId>(rpcRequest.query_id());
 
         TAbortQueryOptions options;
+        if (rpcRequest.has_query_tracker_stage()) {
+            options.QueryTrackerStage = rpcRequest.query_tracker_stage();
+        }
```

Comment:
It is interesting and not clear why we don't have this for GetQueryResult: https://github.com/ytsaurus/ytsaurus/pull/825/files#diff-3218387956293ee1e6820ce23c18208bfe2bfa199e70a2ef32b69ca8467c12eaR145

It doesn't even have Options and doesn't accept a stage

Reply:
Just because we don't need them for now. Maybe we have no reason to add everywhere
```
        if (rpcRequest.has_query_tracker_stage()) {
            options.QueryTrackerStage = rpcRequest.query_tracker_stage();
        }
```

Reply:
>Just because we don't need them for now.

For me it was unexpected that this parameter is lost somewhere. I would prefer to have all parameters passed to QT proxy, even if they are unused(like stage in most methods). Just so in the next time (the most realistic case is when we will start storing metadata of queries somewhere) we have it and can start using, without searching where it was lost.

In fact, I only discovered it because I added a test for a different stage. It was very easy to miss without proper tests, which may happen in future

For GetQueryResult it seems fine. It is not possible in code to use stage so it  shouldn't be a source of bugs